### PR TITLE
[w3t, e2e] Wait for input button to not be disabled

### PR DESCRIPTION
**Pull request by geoknee**
I am seeing this issue on production right now. I want to add some message to disambiguate this problem from other "we are stuck waiting on something" puppeteer error messages.

### CODE REVIEW

File: packages/e2e-tests/puppeteer/scripts/web3torrent.ts [hunk 1]
```diff
@@ -31,16 +31,21 @@ export async function uploadFile(
...
   filePath?: string
 ): Promise<string> {
-  await page.waitForSelector('input[type=file]');
-
+  // https://pub.dev/documentation/puppeteer/latest/puppeteer/FileChooser-class.html
+  // Not clear why puppeteer FileChooser won't work out of box. We are doing it manually for now.
+  const inputUploadHandle = await page.waitForSelector('input[type=file]');
+  if (inputUploadHandle.getProperty('disabled')) {
+    console.warn('Input button is disabled!!');
+    await page.waitFor(4000);
```
Comment (snario) [inline packages/e2e-tests/puppeteer/scripts/web3torrent.ts:39]: Couldn't we use `[disabled=false]` in the selector to wait for it being enabled?

Comment (geoknee) [reply packages/e2e-tests/puppeteer/scripts/web3torrent.ts:39]: Maybe, I will try. It might have to be `disabled=undefined`

Comment (geoknee) [reply packages/e2e-tests/puppeteer/scripts/web3torrent.ts]: 6a4086a

Comment (geoknee) [reply packages/e2e-tests/puppeteer/scripts/web3torrent.ts]: Tested with https://chrome.google.com/webstore/detail/css-selector-tester/bbklnaodgoocmcdejoalmbjihhdkbfon

**Pull request merged**